### PR TITLE
Add back visual intersection test

### PR DESCRIPTION
**Pull request by jridgewell**
Sriram [complained](https://github.com/ampproject/amphtml/commit/836aa5ac04cbfd606e9160e29124c88cf0532eb5#commitcomment-18100410).

### CODE REVIEW

**Comment by sriramkrish85** [comment]
@jridgewell Thanks for fixing :) sorry for being annoying :(

File: test/fixtures/served/iframe-intersection.html [hunk 1]
```diff
@@ -1,25 +1,31 @@
...
+        sentinel: 'amp',
+        changes: event.data.changes,
+        type: 'received-intersection'
+      }, '*');
+    }
+  }
+});
+
 function sendPostMessage() {
   ampWindow./*OK*/postMessage({
```
Comment (sriramkrish85) [inline test/fixtures/served/iframe-intersection.html]: this is not defined.

**Comment by sriramkrish85** [comment]
ping

**Comment by jridgewell** [comment]
Are you going to review it? 😛

File: test/fixtures/served/iframe-intersection.html (hunk 1 above)
Comment (jridgewell) [inline test/fixtures/served/iframe-intersection.html]: I expected my update to make this hidden so I didn't comment. Fixed.

**Pull request merged**